Updated XML Samples

Andrew-Bennett opened the pull request:
Updated the XML samples, as currently they don't work and can't be used in their current state.
-Removed Restrictions tag from newPassword & reenterPassword.
-Updated PasswordValidation regex.

azurecla commented:
Hi __@MandiOhlinger__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

MandiOhlinger commented:
#sign-off

davidwrede commented:
@MandiOhlinger, seems like this is out-of-date. Can I close this PR?

MandiOhlinger commented:
Yes. Closing PR.

Andrew-Bennett committed:
Updated XML Samples

Updated the XML samples, as currently they don't work and can't be used in their current state.
-Removed Restrictions tag from newPassword & reenterPassword.
-Updated PasswordValidation regex.
